### PR TITLE
Fix format of position info WIP

### PR DESCRIPTION
**Pull request by Pengpengwanga**
Signed-off-by: Pengpeng Wang <pengpengw@vmware.com>

### CODE REVIEW

File: poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java [hunk 1]
```diff
@@ -716,6 +686,45 @@ public void saveSensorAssetsToFlowgate(Map<String, Asset> exsitingSensorAssets,
       return pduAssets;
    }
 
+   public String getPositionInfo(Asset asset) {
```
Comment (yixingjia) [inline poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java:689]: getSensorPositionInfo

File: poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java [hunk 2]
```diff
@@ -716,6 +686,45 @@ public void saveSensorAssetsToFlowgate(Map<String, Asset> exsitingSensorAssets,
...
+      if(rackUnitNumber != 0) {
+         rackUnitInfo = FlowgateConstant.RACK_UNIT_PREFIX  + rackUnitNumber;
+         positionInfo.append(rackUnitInfo);
+         if(sensorAssetJustfication == null || sensorAssetJustfication.isEmpty()) {
+            return positionInfo.toString();
+         }
+         String sensorInfo = sensorAssetJustfication.get(FlowgateConstant.SENSOR);
+         try {
+            Map<String,String> sensorInfoMap = mapper.readValue(sensorInfo, new TypeReference<Map<String,String>>() {});
+            positionFromAsset = sensorInfoMap.get(FlowgateConstant.POSITION);
```
Comment (yixingjia) [inline poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java:706]: what if the positionFromAsset not exsit?

Comment (Pengpengwanga) [reply poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java:706]: done

File: poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java (hunk 1 above)
Comment (Pengpengwanga) [reply poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java]: Done

**Pull request merged**